app/models.py

```diff
@@ -28,18 +28,18 @@ class QuizTimeLimit(BaseModel):
 
 
 class QuestionMetadata(BaseModel):
-    grade: str
-    subject: str
-    chapter: str
-    topic: str
-    competency: List[str]
-    difficulty: str
+    grade: Optional[str]
+    subject: Optional[str]
+    chapter: Optional[str]
+    topic: Optional[str]
+    competency: Optional[List[str]]
+    difficulty: Optional[str]
```

Comment:
Don't we have to put in `= None` as well like done in classes above?

Reply:
Don't know. It worked on local without it.

I assumed `None` is just the default value if no value is set. Here if no value is set and the field itself is absent, that's okay. Unless I understand wrong!